packages/block-editor/src/store/selectors.js

```diff
@@ -1757,12 +1757,26 @@ export const __experimentalGetAllowedBlocks = createSelector(
 	]
 );
 
+export const __experimentalGetParsedBlocks = createSelector(
```

Comment:
`why is this a selector if it doesn't use "state"?

Reply:
I had a prototype that used the state but then I ditched it. I left it here because `createSelector` creates a memoried function. I think we can replace it with `_.memoize`? 🤔

Reply:
I wonder if we can remove this function entirely. Why is it needed if the  selector using it is already memoized `__experimentalGetParsedPatterns` ?

Reply:
Calling `__experimentalGetParsedPatterns` parses all the patterns at once. I use this function to break up the task into units.